jdi-cucumber-test-generator-integration-tests/src/test/resources/com/epam/test_generator/api/crud/Registration.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Registration
+
+  Scenario: registration
```

Comment:
I guess there should be no steps in `given`. 
`When I register a new user
Then User is created`